src/inversify.config.ts

```diff
@@ -29,11 +30,13 @@ export function initContainer(): Container {
   container.bind<string>(TYPES.CHS_COMPANY_PROFILE_API_LOCAL_URL).toConstantValue(getEnvOrThrow('CHS_COMPANY_PROFILE_API_LOCAL_URL'))
   container.bind<Optional<string>>(TYPES.PIWIK_SITE_ID).toConstantValue(getEnv('PIWIK_SITE_ID'))
   container.bind<Optional<string>>(TYPES.PIWIK_URL).toConstantValue(getEnv('PIWIK_URL'))
+  container.bind<Optional<string>>(TYPES.DISSOLUTIONS_API_URL).toConstantValue(getEnv('DISSOLUTIONS_API_URL'))
```

Comment:
Feels like this shouldn't be optional. If it isn't there, our app won't work and should fail early